include/emulator/disk.h

```diff
@@ -14,7 +14,7 @@ typedef struct Disk {
 
 bool diskEvent(Disk* pDisk, s32 nEvent, void* pArgument);
 
-extern _XL_OBJECTTYPE gClassDisk;
```

Comment:
accidental change? at the very least seems unrelated

Reply:
this is intended, in disk.c the class is named ``gClassDD``, I wasn't sure which one to use tbh so I can revert this to ``gClassDisk`` and fix the definition instead

Reply:
oh I see. I thought you were renaming it but just fixing it makes sense